src/features/query-bar/__tests__/FilterButton-test.js

```diff
@@ -22,6 +25,33 @@ describe('feature/query-bar/components/filter/FilterButton', () => {
             const Button = wrapper.find('Button');
             expect(Button.props().isDisabled).toEqual(false);
         });
+
+        test('Should close the menu and empty out transientConditions when Apply button is clicked', () => {
+            const wrapper = getWrapper({ conditions: [] });
+            wrapper.instance().setState({
+                transientConditions: validConditions,
+                isMenuOpen: true,
+            });
+
+            wrapper.find('PrimaryButton').simulate('click');
+
+            const Flyout = wrapper.find('Flyout');
+            expect(Flyout.props().overlayIsVisible).toBe(false);
```

Comment:
.toBeFalsy()

This is a useful doc to browse -> https://jestjs.io/docs/en/expect